pkg/operator/deploy/helpers.go

```diff
@@ -133,28 +133,17 @@ func getMeteringAnsiblePath(manifestDir, platform string) (string, error) {
 	return ansibleOperatorManifestDir, nil
 }
 
-// Resource contains information about an individual resource that metering manages
-type Resource struct {
-	UseAbsolutePath  bool
-	SkipYAMLDecoding bool
-	Path             string
-	Resource         interface{}
-}
-
-// InitObjectFromManifest is the driver function for building up a path to the metering-ansible-operator directory
-// from the base @manifestDir directory and initializing the metering resource/CRD objects from the YAML manifests.
-func InitObjectFromManifest(manifestDir, meteringConfigCRFile string, cfg *Config) error {
-	var resources MeteringResources
+func ReadMeteringAnsibleOperatorManifests(manifestDir, platform string) (*OperatorResources, error) {
+	var resources OperatorResources
 
-	ansibleOperatorManifestDir, err := getMeteringAnsiblePath(manifestDir, cfg.Platform)
+	ansibleOperatorManifestDir, err := getMeteringAnsiblePath(manifestDir, platform)
 	if err != nil {
-		return fmt.Errorf("Failed to get the path to the metering-ansible-operator directory: %v", err)
+		return nil, fmt.Errorf("Failed to get the path to the metering-ansible-operator directory: %v", err)
 	}
 
 	pathToCRDMap := map[string]string{
 		"hiveTable":        hivetableFile,
 		"prestoTable":      prestotableFile,
-		"meteringConfig":   meteringconfigFile,
```

Comment:
this is the metering config CRD, not the CR file

Reply:
In general, these names might need to be updated to be more verbose.

Reply:
hah oops